src/types/ButtonProps.ts

```diff
@@ -1,6 +1,7 @@
 interface ButtonProps {
-    disabled: boolean;
     title: string;
+    disabled?: boolean;
+    bgColor?: string;
```

Comment:
Not sure about this solution, looks like it's coupling code and styling, I think style should go always just in css and code or business related things should go in the ts/tsx part, makes sense to have title, enabling and links in the button properties but maybe it's better to have styling separated.